packages/gatsby/src/pages/index.js

```diff
@@ -191,27 +196,27 @@ const IndexPage = ({data, searchState, onSearchStateChange}) => {
           <Section>
             <SectionContent>
               <SellingPoints>
-                <SellingPoint imgUrl={agendaIcon}>
+                <SellingPoint imgUrl={agendaIcon} url="/features/workspaces">
                   <h3>Workspaces</h3>
                   Split your project into sub-components kept within a single repository.
                 </SellingPoint>
-                <SellingPoint imgUrl={laptopIcon}>
+                <SellingPoint imgUrl={laptopIcon} url="/configuration/yarnrc#yarnUpgradePath">
                   <h3>Stability</h3>
                   Yarn guarantees that an install that works now will continue to work the same way in the future.
                 </SellingPoint>
-                <SellingPoint imgUrl={noteIcon}>
+                <SellingPoint imgUrl={noteIcon} url="/advanced/contributing">
                   <h3>Documentation</h3>
                   Special care is put into our documentation, and we keep improving it based on your feedback.
                 </SellingPoint>
-                <SellingPoint imgUrl={puzzleIcon}>
+                <SellingPoint imgUrl={puzzleIcon} url="/features/plugins">
                   <h3>Plugins</h3>
                   Yarn cannot solve all your problems - but it can be the foundation for others to do it.
                 </SellingPoint>
-                <SellingPoint imgUrl={rocketIcon}>
+                <SellingPoint imgUrl={rocketIcon} url="/features/pnp">
                   <h3>Innovation</h3>
                   We believe in challenging the status quo. What should the ideal developer experience be like?
                 </SellingPoint>
-                <SellingPoint imgUrl={worldwideIcon}>
+                <SellingPoint imgUrl={worldwideIcon} url="https://github.com/yarnpkg/berry">
```

Comment:
I'm not 100% confident that each item points to the most relevant page

Reply:
GitHub is perfect for openness 👍